lib/config.js

```diff
@@ -369,17 +376,40 @@ function readEnv () {
 }
 
 function normalize (opts) {
-  normalizeIgnoreOptions(opts)
   normalizeKeyValuePairs(opts)
   normalizeNumbers(opts)
   normalizeBytes(opts)
   normalizeArrays(opts)
   normalizeTime(opts)
   normalizeBools(opts)
+  normalizeIgnoreOptions(opts)
   truncateOptions(opts)
```

Comment:
Move `normalizeIgnoreOptions` down in the list in order to ensure `normalizeArrays` had run, and that the `transactionIgnoreUrls` value would already be converted from a comma,delimited,string into an array (if necessary)